### PR TITLE
release v0.5.3

### PR DESCRIPTION
**Pull request by ubergesundheit**
:wrench: Please make sure to familiarize yourself with [Development section of README.md](../README.md#Development) before releasing a PR to this repository.

:warning: If you want to **create a new release** of this orb please read [Releases section of README.md](../README.md#Releases) carefully :exclamation:

### Checklist

- [ ] Update the [appcatalog docs](https://github.com/giantswarm/giantswarm/blob/master/areas/managed-services/adding_app_to_appcatalog.md) to use the new release.

### CODE REVIEW

**Comment by ubergesundheit** [comment]
Tested here: https://circleci.com/gh/giantswarm/ignition-operator/248

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.3] 2020-02-11
```
Comment (TheoBrigitte) [inline CHANGELOG.md]: You also need to update the links section at the end of this file (next to `[0.5.2]`), to add links for the upcoming release.

Comment (ubergesundheit) [reply CHANGELOG.md]: :+1:

Comment (TheoBrigitte) [reply CHANGELOG.md]: Check https://github.com/giantswarm/architect-orb/pull/62/files

**Comment by ubergesundheit** [comment]
Sorry, I didn't test this properly and will send a reverting PR. See https://app.circleci.com/jobs/github/giantswarm/ignition-operator/250

**Pull request closed without merging**